### ISSUE TITLE
[BUG] Unhandled arguments cause program to fail on launch

### ISSUE THREAD

**Issue by NICHOLAS85** [Merrit/nyrna#42]
**Describe the bug**
If you were to pass an argument that isn't currently handled by Nyrna you'll end up with a blank application window and an error in your console.

**To Reproduce**
Pass an argument to Nyrna and view its output. For example `nyrna --help`

**Expected behavior**
For Nyrna to exit gracefully with an error message indicating that argument is unknown

**Screenshots**
![image](https://user-images.githubusercontent.com/15698885/124142816-919e9b00-da58-11eb-8229-bfbff3a2326c.png)

```cc
$ ./Nyrna-latest-x86_64.AppImage --help
Gtk-Message: 10:31:34.778: Failed to load module "colorreload-gtk-module"
Gtk-Message: 10:31:34.778: Failed to load module "window-decorations-gtk-module"
Gtk-Message: 10:31:34.779: Failed to load module "appmenu-gtk-module"
flutter: Unknown argument: FormatException: Could not find an option named "help".
[ERROR:flutter/lib/ui/ui_dart_state.cc(199)] Unhandled Exception: NoSuchMethodError: The method 'wasParsed' was called on null.
Receiver: null
Tried calling: wasParsed()
#0      ArgumentParser._checkToggleFlag (package:nyrna/arguments/argument_parser.dart:61)
#1      ArgumentParser._parse (package:nyrna/arguments/argument_parser.dart:46)
#2      ArgumentParser.init (package:nyrna/arguments/argument_parser.dart:21)
#3      parseArgs (package:nyrna/main.dart:34)
#4      main (package:nyrna/main.dart:19)
#5      _runMainZoned.<anonymous closure>.<anonymous closure> (dart:ui/hooks.dart:140)
#6      _rootRun (dart:async/zone.dart:1354)
#7      _CustomZone.run (dart:async/zone.dart:1258)
#8      _runZoned (dart:async/zone.dart:1789)
#9      runZonedGuarded (dart:async/zone.dart:1777)
#10     _runMainZoned.<anonymous closure> (dart:ui/hooks.dart:138)
#11     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:283)
#12     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:184)
```

**Software (please complete the following information):**
 - OS: `Bedrock Linux 0.7.20 Poki, Arch and Debian strata`
 - Destop Environment: `KDE 5.20.5`

Great work on Nyrna btw! Quite a handy tool

**Comment by Merrit** [comment]
Thanks for the issue! Can confirm on my end & will queue for a fix 👍

**Comment by Merrit** [comment]
Thanks again for the report 😄 

This is fixed with b1db01a

The fix is available as a pre-release https://github.com/Merrit/nyrna/releases/tag/v2.1.0-beta.3
